readme.md

```diff
@@ -35,11 +35,11 @@ GitHub username.
 
 #### token
 
-Type: `string`  
```

Comment:
Our messages crossed. The linebreaks have been restored. In your original readme, some type: `string` have a linebreak, some don't. I was careful to do it exactly like yours. I was used to having my editor change a bunch of stuff on save, now that is disabled.